lib/matplotlib/figure.py

```diff
@@ -1728,6 +1730,10 @@ def tight_layout(self, renderer=None, pad=1.08, h_pad=None,
                                          rect=rect)
 
         self.subplots_adjust(**kwargs)
+        self._tight_parameters.update(
+                (k, v) for k, v in six.iteritems(
```

Comment:
Just do this with a tuple of tuples `(('pad', pad), ('h_pad', h_pad)...)`.  I think that will be a bit more readable and get rid of the `six` usage.